frontend/src/components/Icons/Home.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Home = ({ color }) => (
+  <svg width="24" height="24" viewBox="0 0 24 24" fill="#f2f2f2" xmlns="http://www.w3.org/2000/svg">
+    <path d="M3 9L12 2L21 9V20C21 20.5304 20.7893 21.0391 20.4142 21.4142C20.0391 21.7893 19.5304 22 19 22H5C4.46957 22 3.96086 21.7893 3.58579 21.4142C3.21071 21.0391 3 20.5304 3 20V9Z" stroke={`#${color}`} stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+    <path d="M9 22V12H15V22" stroke={`#${color}`} stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+  </svg>
+);
+
+Home.defaultProps = {
+  color: 'FF7F48',
+}
+
```

Comment:
Please check this out https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs. Then remove this component and the Inbox component and import both the svgs into the View Component. 
You can then change fill colors normally via css `.Tab-active svg path { fill: $primary-color; }`